packages/wonder-blocks-color/color-with-name.js

```diff
@@ -0,0 +1,50 @@
+// @flow
```

Comment:
moved from docs.md

Reply:
Could you move these helper files into a sub-directory? Perhaps just one named `docs` or `docutils`?

Reply:
Good idea.  I'll move these to `docutils`.